packages/react/src/components/Tabs/Tabs-test.js

```diff
@@ -197,33 +205,40 @@ describe('Tabs', () => {
 
       const firstTab = wrapper.find('.firstTab').last();
       const lastTab = wrapper.find('.lastTab').last();
-      const leftKey = 37;
-      const rightKey = 39;
-      const spaceKey = 32;
-      const enterKey = 13;
 
       describe('state: selected', () => {
         it('updates selected state when pressing arrow keys', () => {
-          firstTab.simulate('keydown', { which: rightKey });
+          firstTab.simulate('keydown', { ...ArrowRight });
```

Comment:
```suggestion
          firstTab.simulate('keydown', ArrowRight);
```

If you want! Although I'm sure you had a good reason for the spread 👍

Reply:
just to get all of the properties in the simulated event and not just `which`

Reply:
@emyarod not sure I understand, it should have all of them? https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/internal/keyboard/keys.js#L72-L76